geode-core/src/main/java/org/apache/geode/management/internal/configuration/validators/MemberValidator.java

```diff
@@ -52,16 +55,27 @@ public void validateCreate(AbstractConfiguration config, CacheConfigurationManag
     if (existingElementsAndTheirGroups.size() == 0) {
       return;
     }
+
+    // if the configuration is not groupable and already exists, throw exception
+    if (!(config instanceof GroupableConfiguration)) {
+      throw new EntityExistsException(
+          config.getClass().getSimpleName() + " '" + config.getId()
+              + "' already exists");
+    }
+
+    // if configuration is groupable, then check if it's already in the group
     String configGroup = AbstractConfiguration.getGroupName(config.getGroup());
     if (existingElementsAndTheirGroups.keySet().contains(configGroup)) {
       throw new EntityExistsException(
           config.getClass().getSimpleName() + " '" + config.getId()
               + "' already exists in group " + configGroup);
     }
 
+    // if other group and this new group has common members, then throw exception
+    String[] groups = existingElementsAndTheirGroups.keySet().toArray(new String[0]);
     Set<DistributedMember> membersOfExistingGroups =
-        findServers(existingElementsAndTheirGroups.keySet().toArray(new String[0]));
-    Set<DistributedMember> membersOfNewGroup = findServers(config.getGroup());
+        findMembers(false, groups);
```

Comment:
I liked the old findServers(groups) instead of findMembers(false, groups). Can we bring it back and have it call findMembers(false, groups)?